app/src/main/kotlin/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -1662,14 +1663,26 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
         if (presenter.isAlwaysShowFirstViewOnAppStartEnabled() &&
             LifecycleHandler.isAppInBackground()
         ) {
-            // Pattern matches urls which are NOT allowed to show the first view after app is started
-            // This is
-            // /config/* as these are the settings of HA but NOT /config/dashboard. This is just the overview of the HA settings
-            // /hassio/* as these are the addons section of HA settings.
+            /**
```

Comment:
Why change this to KDoc style? The main advantages (info in tooltips, formatting, generated docs) don't apply anyway inside code blocks, and it is inconsistent with other code.

Reply:
Will rollback